packages/gatsby-theme-docs/src/layouts/internals/layout-application.js

```diff
@@ -36,12 +37,26 @@ const Container = styled.div`
 `;
 
 const LayoutApplication = (props) => (
-  <UiKitThemeProvider theme={designSystem.uikitTheme}>
+  <UiKitThemeProvider
+    theme={{
+      ...designSystem.uikitTheme,
+      // Docs specific theme properties
+      colors: {
+        light: {
+          primary:
+            props.websitePrimaryColor || designSystem.colors.light.primary,
```

Comment:
https://github.com/commercetools/commercetools-docs-kit/blob/nm-website-primary-color/packages/ui-kit/src/design-system.js#L18  is not the default color sven posted in the issue ( #00B39E ).  Did you intend to not change it or is the intention to change it separately ( maybe during #226 ?).

Reply:
I guess the question is, should there be a default color or should we make the `websitePrimaryColor` option required? So that each website must specify the color.

@Shecki @nkuehn

Reply:
@nkuehn how many "colors" do we need in the beginning?
as a fallback, i think we should use the "key-Color"

it is not allowed to use "any-color" i would like to come up with a Selection of Colors.
First, i would use the most "prominent ones from our UI-KIT library.

Reply:
So should there be a default primary color in the theme?

- [ ] yes
- [ ] no

If so, is it the `#00B39E`?

Reply:
from my side, for the beginning yes.
Later i will look at all Products/Solutions and define the color.
As the colors have to be used in Banners/Graphics as well

Reply:
Ok thanks